### DISCUSSION TITLE
What is holding up the next release?

### DISCUSSION THREAD

**Discussion by davidwarford** [google/filament#3443] [General]
currently I am having to use the nightly version because of some breaking changes in google chrome. But it's been months I have been waiting to upgrade, what is the hold up?

**Comment by romainguy** [comment]
We're holding off on tagging a new release until we can finish an internal migration at Google. We should hopefully be done soon. New releases are usually cut on Mondays and because of said migration we haven't been adding anything major, mostly doing bug fixes, and as such the nightly builds should be fine right now.

**Reply by davidwarford** [reply]
Is the future of filament project in any sort of uncertainty? Is this a technical migration or a migration of people between departments? If you can't answer that's fine, I'm just concerned because I'm building something for my business that uses filament and would hate to see it lose support.

**Reply by romainguy** [reply]
I guess migration was the wrong term here. Filament development happens on GitHub but internal teams at Google use a monorepo called "google3" which is very, very different in every way. We're helping internal users of Filament setup a more automated and more regular merge of Filament to that monorepo. To make this happen we need to help solve a few technical problems (for instance to run automated Filament tests on machines without GPUs, fix bugs specific to how Filament is used by Google apps, etc.).

**Reply by davidwarford** [reply]
Oh okay, great to hear their is no danger of the project going away (or at least immediate danger)
